Unable to run on windows 10

Hi

I'm using size-limit 9 with NodeJs 18 and windows 10 

Using powershell I have de following error.

```powershell
C:\desarrollos\ActiveDirectory>npm run size --why
npm ERR! No workspaces found:
npm ERR!   --workspace=--usage
```

The second option y run at WSL but I have one annoying bug that I can solve

```
⠋ Running JS in headless ChromeError: Failed to launch the browser process! spawn C:\Program Files\Google\Chrome\Application\chrome.exe ENOENT
```

It's possible to run `npm run size --why` and saving the result, without open the browser?

--- a/packages/size-limit/load-plugins.js
+++ b/packages/size-limit/load-plugins.js
@@ -1,6 +1,3 @@
-import { createRequire } from 'node:module'
-const require = createRequire(import.meta.url)
-
 function toArray(obj) {
   return typeof obj === 'object' ? Object.keys(obj) : []
 }
@@ -26,13 +23,7 @@ export default async function loadPlugins(pkg) {
       .concat(toArray(pkg.packageJson.devDependencies))
       .concat(toArray(pkg.packageJson.optionalDependencies))
       .filter(i => i.startsWith('@size-limit/') || i.startsWith('size-limit-'))
-      .map(i =>
-        import(
-          require.resolve(i, {
-            paths: [process.cwd()]
-          })
-        ).then(module => module.default)
-      )
+      .map(i => import(i).then(module => module.default))
   ).then(arr => arr.flat())
 
   return new Plugins(list)

